.travis.yml

```diff
@@ -2,20 +2,44 @@ language: python
 python: 3.7
 sudo: true
 dist: bionic
-cache: pip
+
+cache:
+  directories:
+    - "$HOME/.cache/pip"
+    - "$HOME/google-cloud-sdk/"
+
 before_cache:
   - chown -R travis:travis $HOME/.cache/pip
 
+env:
+  global:
+    - GOOGLE_DOCKER_IMAGE=gcr.io/athenian-1/api
+
 install:
   - cd server
   - pip install -r requirements.txt
   - pip install -r lint-requirements.txt
   - pip install -r test-requirements.txt
   - pip install -e .
 
-script:
-  - flake8
-  - pytest
+stages:
+  - name: test
+  - name: deploy
+    if: branch = add-cd AND type != pull_request
```

Comment:
Shouldn't this be an OR?

Reply:
it's WIP, here should be `if: branch = add-cd AND type != pull_request`
should be and because it will trigger deploy in the case of PR to pool request, please correct me if I'm wrong

Reply:
ah ok, I got it now. thanks!